src/analysis/calculus/times_cont_diff.lean

```diff
@@ -1315,9 +1315,9 @@ begin
     have : f = e.symm ∘ (e ∘ f),
       by { ext y, simp only [function.comp_app], rw e.symm_apply_apply (f y) },
     rw this,
-    exact H.continuous_linear_map_comp _ },
+    exact H.continuous_linear_map_comp (e.symm : G →L[𝕜] F) },
```

Comment:
Why did these lines have to change?

Reply:
I think I had picked the "wrong" `has_coe_to_fun` instance.  The `continuous_linear_map_comp` lemma is a bit fragile since it needs to do higher-order matching if you don't give the function explicitly.  I've changed the proofs back now.